### PR TITLE
chore: update expected bidi errors

### PR DESCRIPTION
**Pull request by OrKoN**
We can consider this errors coming from Firefox's BiDi implementation as errors that require rerunning the wait tasks. We need to replace it with the standard message once https://github.com/w3c/webdriver-bidi/issues/540 lands.

### CODE REVIEW

File: packages/puppeteer-core/src/common/WaitTask.ts
Comment (whimboo) [inline packages/puppeteer-core/src/common/WaitTask.ts]: I would include the `AbortError: ` part here to really only let it apply for IPC issues when trying to send the response in Firefox.

**Review by Lightning00Blade — APPROVED** [review]

**Comment by OrKoN** [comment]
Looks like there could be more variations in errors. Let's wait till the error has been spec'ed and well defined.

File: packages/puppeteer-core/src/common/WaitTask.ts
Comment (Lightning00Blade) [inline packages/puppeteer-core/src/common/WaitTask.ts:241]: There is the same message a few line above with a comment, pointing to the same issue.  
Let's close this PR.

**Review by Lightning00Blade — APPROVED** [review]

**Comment by OrKoN** [comment]
@whimboo mentioned that he will take a look at the failing tests

**Comment by whimboo** [comment]
> @whimboo mentioned that he will take a look at the failing tests

Am I supposed to apply this patch for further investigation, or is this PR blocked on my findings? If it's not blocking I would wait until this PR has been landed.

**Pull request merged**
